client/App/pages/LandingPage/components/Dropdown/Dropdown.module.scss

```diff
@@ -0,0 +1,30 @@
+.dropdown {
+  background-color: #f6f6f6;
+  width: 100%;
+  padding: 45px 0;
+
+  &__list {
```

Comment:
I can see that those class names after bundling are unreadable please add those line to our webpack config. `    test: /\.scss$/,
        use: [
          'style-loader',
          {
            loader: 'css-loader',
            options: {
              modules: {
                localIdentName: '[local]__[hash:base64:5]',
              },
            },
          },
          'sass-loader',
        ],`

Reply:
Right now we've got smth like this:
`      {
        test: /\.scss$/,
        use: ['style-loader', 'css-loader', 'sass-loader'],
      }
`
Should i replace this test or only update with a new configuration?

Reply:
My example is extended version of our, there for replace it